source/fundamentals/indexes.txt

```diff
@@ -306,7 +306,7 @@ the geospatial data for inclusion, intersection, and proximity. For more informa
 :manual:`Geospatial Queries </geospatial-queries/>`.
 
 To create a ``2dsphere`` index, you must specify a field that contains only **GeoJSON objects**. For more details on this
-type, see the MongoDB server manual page on :manual:`GeoJSON objects </reference/geojson>`.
+type, see the MongoDB Server manual page on :manual:`GeoJSON objects </reference/geojson>`.
```

Comment:
[suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

Reply:
This looks okay to me...?